### PR TITLE
Add three new tutorials and small tweaks to docs

### PR DESCRIPTION
**Pull request by FabioRosado**
This PR does a few things:

- Adds a tutorial for [[fetch]]
- Adds a tutorial for [[runtimes]]
- Add a 404 error template (its in markdown so it has the docs theme and users can use the search bar)
- Small tweaks to references
- Removes the "Coming soon" pages (deployment, setup)

### CODE REVIEW

File: docs/error.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Not found!
+
+The page that you looked for could not be found. If you think this is a mistake, please [open an issue](https://github.com/pyscript/pyscript/issues/new/)
```
Comment (JeffersGlass) [inline docs/error.md:3]: Nice!

File: docs/reference/elements/py-config.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ The `<py-config>` element should be placed within the `<body>` element.
   lang = "python"
 </py-config>
 ```
-Note: `[[runtimes]]` is a TOML table. Make sure this is the last item within a py-config, as the properties created after it go into the runtimes object.
+
+```{note}
+`[[runtimes]]` is a TOML table. Make sure this is the last item within a py-config, as the properties created after it go into the runtimes object.
+```
```
Comment (JeffersGlass) [inline docs/reference/elements/py-config.md:30]: Technically, it's just that TOML tables have to go after top level keys. So you can do:
```
[[runtimes]]
...
[[fetch]]
...
you just can't have the top level keys after either.

Comment (FabioRosado) [reply docs/reference/elements/py-config.md:30]: That's a great point, I just turned the note into a proper note haha
Do you think we should remove the note? 🤔

**Review by JeffersGlass — APPROVED** [review]
LGTM! Does make me think we should have some docs for `Element` though... but that's for another day.

**Comment by FabioRosado** [comment]
Yeah that's a good point, I'll add it to my queue to start writing something about `Element` 😄  Also thank you for creating the issue!

**Pull request merged**